cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Poll.hs

```diff
@@ -4,7 +4,10 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 
 module Cardano.CLI.EraBased.Run.Governance.Poll
-  ( runGovernancePollCmds
+  ( runGovernancePollCmds,
+    runGovernanceCreatePoll, {- For legacy caller only -}
+    runGovernanceAnswerPoll, {- For legacy caller only -}
+    runGovernanceVerifyPoll  {- For legacy caller only -}
```

Comment:
We don't need to add a comment here.  It is okay to export run commands even if no-one uses them.

In fact, it is preferred in case we want to change the tests to use these instead of the CLI.

Reply:
Changed :+1: